server/routes/referrals/referralsController.test.ts

```diff
@@ -67,19 +67,26 @@ describe('GET /referrals/start', () => {
 })
 
 describe('POST /referrals/start', () => {
-  describe('when searching for a CRN found in Delius', () => {
+  describe('when searching for a CRN found in Delius and an intervention has been selected', () => {
     beforeEach(() => {
       communityApiService.getServiceUserByCRN.mockResolvedValue(deliusServiceUser.build())
     })
 
     it('creates a referral on the interventions service and redirects to the referral form', async () => {
+      const currentlyHardcodedInterventionId = '24b8cd48-ccb2-48e0-a842-f5673c73e38f'
```

Comment:
please can you use `98a42c61-c30f-4beb-8062-04033c376e2d` for this for now? it avoids us having to change it in the API test data

Reply:
Sure